File: openapi/openapi.json/code_samples/typescript/iot-connectivity_v1_devices_{iccid}/get.ts
import { TestSdk } from 'test-sdk';

(async () => {
  const testSdk = new TestSdk({
    token: 'YOUR_TOKEN',
  });

  const { data } = await testSdk.connectivity.getDeviceDetails('iccid', {
    xTransactionId: 'X-transaction-id',
  });

  console.log(data);
})();

File: openapi/openapi.json/code_samples/python/iot-connectivity_v1_devices_{iccid}/get.py
from test_sdk import TestSdk, Environment

sdk = TestSdk(
    access_token="YOUR_ACCESS_TOKEN",
    base_url=Environment.DEFAULT.value,
    timeout=10000
)

result = sdk.connectivity.get_device_details(
    iccid="iccid",
    x_transaction_id="X-transaction-id"
)

print(result)